apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/DocumentDetailActivity.java

```diff
@@ -50,10 +57,22 @@
 
     private String mFullErrorContents;
 
+    private String mDocumentContent;
+
+    private String mDocumentDate;
+
+    private boolean mDocumentState;
+
+    private String mDocumentError;
+
+    private boolean mDocumentNullStatus;
+
     private ProgressBar mDetailProgress;
 
     private ListView mListView;
 
+    private MenuItem mRefreshDocument;
```

Comment:
Code analysis warning here.

Please use the Android Studio commit dialog with the following checked box options:

<img width="1356" alt="Screen Shot 2019-05-02 at 3 30 09 PM" src="https://user-images.githubusercontent.com/623672/57110928-32fdfb00-6cef-11e9-986d-6243e1eca1dd.png">

Reply:
Still unresolved. You assign the variable but never use it.